test/smartlead/write-delete/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os/signal"
+	"syscall"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/smartlead"
+	connTest "github.com/amp-labs/connectors/test/smartlead"
+	"github.com/amp-labs/connectors/test/utils"
+)
+
+type campaignPayload struct {
+	Name string `json:"name,omitempty"`
+}
+
+var (
+	objectName = "campaigns" // nolint: gochecknoglobals
+)
+
+func main() {
+	// Handle Ctrl-C gracefully.
+	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer done()
+
+	// Set up slog logging.
+	utils.SetupLogging()
+
+	conn := connTest.GetSmartleadConnector(ctx)
+	defer utils.Close(conn)
+
+	fmt.Println("> TEST Create/Delete campaign")
+	fmt.Println("Creating campaign")
+
+	view := createCampaign(ctx, conn, &campaignPayload{
```

Comment:
It might be good to output this so that we can see that it's actually gone through

Reply:
![image](https://github.com/user-attachments/assets/3cfc4680-dcdc-4917-a985-227a1ff76e6c)